charts/pega/README.md

```diff
@@ -401,6 +401,18 @@ tier:
         - name: MY_ENV_NAME
           value: MY_ENV_VALUE
 ```
+### Custom Annotations for deployment
```

Comment:
### Custom annotations for Pods

You may optionally provide custom [annotations](https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/) for Pods as metadata to be consumed by other tools and libraries. Pod annotations may be specified by using the `podAnnotations` element for a given `tier`.

```yaml
tier:
  - name: my-tier
    podAnnotations:
      <annotation-key>: <annotation-value>
```

Reply:
@taz-mon - any other suggestions?

Reply:
Done

Reply:
looks good!